### ISSUE TITLE
training with more epochs

### ISSUE THREAD

**Issue by oussaifi-majdi** [Deci-AI/super-gradients#1482]
### 💡 Your Question

i trained yolo-nas on my custom dataset for 300 epochs but colab some time restarted , how please do for the training process will start from where it left off, so it will not retrain on the initial dataset again.

**Comment by oussaifi-majdi** [comment]
train_params['resume'] = True
train_params["resume_path"] ='/content/gdrive/MyDrive/Cv/checkpointss92/yolo_nas_faces92/ckpt_latest.pth'
trainer.train(model=model,
              training_params=train_params,
              train_loader=train_data,
              valid_loader=val_data)

**Comment by BloodAxe** [comment]
Or buy a GPU since Colab is not meant for sustained training sessions.
